lib/pbio/drv/usb/stm32_usbd/usbd_desc.c

```diff
@@ -76,14 +87,143 @@ __ALIGN_BEGIN uint8_t USBD_DeviceDesc[USB_LEN_DEV_DESC] __ALIGN_END = {
     HIBYTE(USBD_VID),         /* idVendor */
     LOBYTE(USBD_PID),         /* idVendor */
     HIBYTE(USBD_PID),         /* idVendor */
-    0x00,                     /* bcdDevice rel. 2.00 */
-    0x02,
+    LOBYTE(USBD_VERSION_BCD), /* bcdDevice */
```

Comment:
Every time this number changes, the host computer will think this is a new USB device. So we do want it to be something different from the official LEGO firmware. But I'm not sure we want it to change between Pybricks releases.